api/v1alpha1/alluxiocluster_types.go

```diff
@@ -257,14 +257,24 @@ type AlluxioMonitorSpec struct {
 type EtcdSpec struct {
 	Enabled      *bool             `json:"enabled" yaml:"enabled"`
 	Auth         EtcdAuthSpec      `json:"auth,omitempty" yaml:"auth,omitempty"`
-	Image        EtcdImageSpec     `json:"image,omitempty" yaml:"image,omitempty"`
 	NodeSelector map[string]string `json:"nodeSelector,omitempty" yaml:"nodeSelector,omitempty"`
 	ReplicaCount int               `json:"replicaCount,omitempty" yaml:"replicaCount,omitempty"`
+	Resources    ResourcesSpec     `json:"resources,omitempty" yaml:"resources,omitempty"`
 }
 
 type EtcdAuthSpec struct {
-	Rbac  EtcdAuthRbacSpec  `json:"rbac,omitempty" yaml:"rbac,omitempty"`
-	Token EtcdAuthTokenSpec `json:"token,omitempty" yaml:"token,omitempty"`
+	Client EtcdAuthClientSpec `json:"client,omitempty" yaml:"client,omitempty"`
+	Peer   EtcdAuthPeerSpec   `json:"peer,omitempty" yaml:"peer,omitempty"`
```

Comment:
did u mean to add for .Values.auth.client.enableAuthentication and .Values.auth.peer.enableAuthentication seperately? coz that's two different things, one between etcd-client and etcd, the other one between etcd members.

Reply:
remove peer.authentication as a vanilla version of starting a etcd for alluxio to use doesnt need that